Documentation/ApiOverview/Database/ExpressionBuilder/_RepositoryRight.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Repository;
+
+use TYPO3\CMS\Core\Database\ConnectionPool;
+
+final class MyTableRepository
+{
+    private const TABLE_NAME = 'pages';
+    public function __construct(private readonly ConnectionPool $connectionPool) {}
+
+    public function demonstrateRight(): void
+    {
+        $queryBuilder = $this->connectionPool
+            ->getQueryBuilderForTable(self::TABLE_NAME);
+        $expression1 = $queryBuilder->expr()->right(
+            6,
+            $queryBuilder->quote('some-string'),
+        );
+
+        $expression2 = $queryBuilder->expr()->right(
+            '6',
+            $queryBuilder->quote('some-string'),
+        );
+
+        $expression3 = $queryBuilder->expr()->right(
```

Comment:
IMO this example may be redundant, as the "right" method already does this casting internally? Maybe instead another example of a sub expression, like concatenating or so?!

Reply:
No idea, took it from the changelog

Reply:
Also showcases a sub-expression so I left it and adjusted the example.